Documentation/Security/GuidelinesIntegrators/InstallTool.rst

```diff
@@ -150,5 +150,22 @@ information, cache content, etc. and you should clear all caches after
 you changed this value in order to force the rebuild of this data with
 the new encryption key.
 
-Keep in mind that this string is security-related and you should keep
-it in a safe place.
+..  attention::
+    Keep in mind that this string is security-related and you should keep
+    it in a safe place.
+
+The encryption key should be a random hexadecimal key of length 96. You can 
+for example create it with openssl:
+
+..  code-block:: bash
+
+    openssl rand -hex 48 
+    
+From withing TYPO3 it is possible to generate it via API:
+
+
+..  code-block:: php
+
+    use  \TYPO3\CMS\Core\Crypto\Random;
```

Comment:
```suggestion
    use \TYPO3\CMS\Core\Crypto\Random;
```

Reply:
I agree with you (see Slack) we should provide a command for that, a button in the backend, or at least a small PHP snippet.

Reply:
But for now this is better then nothiing?